daaltk-core/src/test/scala/org/trustedanalytics/daaltk/models/dimensionality_reduction/PrincipalComponentsModelTest.scala

```diff
@@ -69,7 +69,7 @@ class PrincipalComponentsModelTest extends TestingSparkContextWordSpec with Matc
       assert(thrown.getMessage.contains("column bogus was not found"))
 
       // invalid k (greater than number of observation columns
-      thrown = the[Exception] thrownBy PrincipalComponentsModel.train(frame, List("1", "bogus", "3", "4", "5", "6"), true, Some(8))
```

Comment:
This was a bug in a test (unrelated to PCA predict), that I just noticed and fixed